installer/bundle/bundle_skel.sh

```diff
@@ -881,7 +887,8 @@ case "$installMode" in
             # Install SCX (and OMI)
             [ -n "${forceFlag}" ] && FORCE="--force" || FORCE=""
             [ -n "${debugMode}" ] && DEBUG="--debug" || DEBUG=""
-            ./bundles/${SCX_INSTALLER} --install $FORCE $DEBUG
+            [ -n "${enableOMFlag}" ] && ENABLE_OM="--enable-opsmgr" || ENABLE_OM=""
+            ./bundles/${SCX_INSTALLER} --install $FORCE $DEBUG $ENABLE_OM
```

Comment:
```
commit 056ebea
Author: Vishal Mishra <vimish@microsoft.com>
Date:   Fri Mar 24 00:20:32 2017
```

This was super easy to find with `git log -p installer/bundle/bundle_skel_Linux.sh`